js/common.js

```diff
@@ -0,0 +1,28 @@
+export const serverURL = "https://js-todo-list-9ca3a.df.r.appspot.com";
+/** structure of todo elements in server.
+ * - 'name': string. name of user.
+ * - 'todoList': array. list of todos.
+ *   - '_id': string. id of todo.
+ *   - 'contents': string. text of todo.
+ *   - 'priority': string. priority of todo. it has 'NONE', 'FIRST', 'SECOND'
+ *   - 'isCompleted': string. status of todo.
+ * - '_id': string. id of user.
+ */
+
+// 할 일이 몇 개 있는지 출력하는 텍스트(총 n 개)를 업데이트하는 로직.
+export function updateCountText() {
+  const todoListCountText = document.querySelector("span.todo-count strong");
```

Comment:
```suggestion
  const todoListCountText = document.querySelector(".todo-count > strong");
```
querySelector를 사용하시면 위와 같은 코드로도 파싱이 가능합니다!

Reply:
CSS 셀렉터에 좀 더 익숙해지면 querySelector를 사용하는데 도움이 많이 되겠군요! 감사합니다.